msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java

```diff
@@ -87,17 +85,16 @@ void acquireToken(@NonNull final Activity activity,
     IAuthenticationResult acquireTokenSilent(@NonNull final AcquireTokenSilentParameters acquireTokenSilentParameters) throws InterruptedException, MsalException;
 
     /**
-     * Returns the PublicClientConfiguration for this instance of PublicClientApplication
-     * Configuration is based on the defaults established for MSAl and can be overridden by creating the
-     * PublicClientApplication using {@link PublicClientApplication#PublicClientApplication(Context, PublicClientApplicationConfiguration)}
+     * Returns the PublicClientConfiguration for this instance of PublicClientApplication.
```

Comment:
nit: do we want to keep the default configuration part?

Reply:
I axed it -- I think this is covered more clearly in our Wiki and the constructor call mentioned here doesn't exist anymore (replaced by factory method)